<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>midisnoop</string>
  </property>
  <widget class="QTableView" name="QTV_MidiMsg">
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::CurrentChanged|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="addAction"/>
    <addaction name="QA_ClearMsgMon"/>
    <addaction name="separator"/>
    <addaction name="configureAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="quitAction"/>
   <addaction name="separator"/>
   <addaction name="addAction"/>
   <addaction name="QA_ClearMsgMon"/>
   <addaction name="separator"/>
   <addaction name="configureAction"/>
   <addaction name="separator"/>
   <addaction name="aboutAction"/>
  </widget>
  <action name="addAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/midisnoop/images/16x16/add.png</normaloff>:/midisnoop/images/16x16/add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add a MIDI message to send.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="QA_ClearMsgMon">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/midisnoop/images/16x16/clear.png</normaloff>:/midisnoop/images/16x16/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear MIDI messages.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/midisnoop/images/16x16/quit.png</normaloff>:/midisnoop/images/16x16/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit this application.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="configureAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/midisnoop/images/16x16/configure.png</normaloff>:/midisnoop/images/16x16/configure.png</iconset>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
   <property name="toolTip">
    <string>Various setup parameters</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/midisnoop/images/16x16/information.png</normaloff>:/midisnoop/images/16x16/information.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About midisnoop.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
